app/src/main/java/org/oppia/android/app/settings/profile/ProfileEditFragmentPresenter.kt

```diff
@@ -151,16 +152,29 @@ class ProfileEditFragmentPresenter @Inject constructor(
         fragment,
         Observer {
           if (it is AsyncResult.Success) {
-            if (fragment.requireContext().resources.getBoolean(R.bool.isTablet)) {
```

Comment:
I don't think you needed to alter the flow. I left the flow intact and only added a few lines of code to help add a Snackbar request to the queue. I have made a gist with [my implementation of the ProfileEditFragmentPresenter](https://gist.github.com/kkmurerwa/c8b1bebf85910cad1c2a0b015e3038da).

Reply:
Okay